libraries/networking/src/NLPacketList.cpp

```diff
@@ -11,15 +11,27 @@
 
 #include "NLPacketList.h"
 
-#include "NLPacket.h"
+#include "udt/Packet.h"
 
-NLPacketList::NLPacketList(PacketType packetType, QByteArray extendedHeader) :
-    PacketList(packetType, extendedHeader)
+NLPacketList::NLPacketList(PacketType packetType, QByteArray extendedHeader, bool isReliable, bool isOrdered) :
+    PacketList(packetType, extendedHeader, isReliable, isOrdered)
 {
+}
+
+NLPacketList::NLPacketList(PacketList&& other) : PacketList(other.getType(), other.getExtendedHeader(), other.isReliable(), other.isOrdered()) {
+    // Update _packets
```

Comment:
We should do a `_packets.reserve(other._packets.size())` here to avoid a bunch of allocations.